zenodo/modules/records/templates/zenodo_records/box/edit.html

```diff
@@ -109,3 +154,45 @@
   </ul>
 </div>
 {%- endif %}
+
+<script>
```

Comment:
Ideally, with the changes discussed IRL, we can completely remove JS since:

- there's no UI validation for the DOI typing
- the action is stored using cookies that expire in 1 week

Reply:
I am doing the cookie check with JS, because if I didn't I would have to modify the landing page view to check for the cookie and it seems excessive to me, let's discuss IRL, maybe I'm forgetting parts of our last discussion.